### PR TITLE
Changes to PaymentMethod documentation

### PR DESCRIPTION
**Pull request by KBbitsP**
Added External Key example to Slate and updated PaymentMethod documentation to point that creating Payment Method has to be done at account level

### CODE REVIEW

File: source/includes/_payment-method.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ All payment operations associated with this payment method will be delegated to
 
 Basic operations to retrieve, list, search and delete payment methods.
 
-Note that the creation of a payment method relies on an operation listed under the [Account resource](https://killbill.github.io/slate/#account-add-a-payment-method). The creation of a Kill Bill PaymentMethod is always associated with a given `Account` and it identifies (through its `pluginName` attribute) the payment plugin that will be used by the system when a payment is made. 
+Note that the creation of a payment method needs to be done on the Account level and the endpoint listed under [Account resource](https://killbill.github.io/slate/#account-add-a-payment-method) must be used for this. The `pluginName` attribute identifies the payment plugin that will be used by the system when this payment method is used for a payment.
```
Comment (reshmabidikar) [inline source/includes/_payment-method.md:33]:
```suggestion
Note that the creation of a payment method needs to be done at the Account level using the [Add a Payment Method](https://killbill.github.io/slate/#account-add-a-payment-method) endpoint. The `pluginName` attribute identifies the payment plugin that will be used by the system when this payment method is used for a payment. 
```

File: source/includes/_account.md [hunk 2]
```diff
@@ -3904,7 +3904,7 @@ curl -v \
     -H "X-Killbill-CreatedBy: demo" \
     -H "X-Killbill-Reason: demo" \
     -H "X-Killbill-Comment: demo" \
-    -d '{ "accountId": "2ad52f53-85ae-408a-9879-32a7e59dd03d", "pluginName": "__EXTERNAL_PAYMENT__"}' \
+    -d '{ "externalKey": "ExternalKey", "accountId": "2ad52f53-85ae-408a-9879-32a7e59dd03d", "pluginName": "__EXTERNAL_PAYMENT__"}' \
     "http://127.0.0.1:8080/1.0/kb/accounts/8785164f-b5d7-4da1-9495-33f5105e8d80/paymentMethods"	
 ```
```
Comment (reshmabidikar) [inline source/includes/_account.md:3909]: Maybe specify a `Request Body` section (See https://killbill.github.io/slate/#subscription-create-a-subscription) and mention that a `Payment Method Resource` needs to be specified? Also add a link to the [Payment Method Resource](https://killbill.github.io/slate/?shell#payment-method-payment-method-resource) section?

**Review by reshmabidikar — COMMENTED** [review]
Added a few minor comments, good to merge once these are addressed.

File: source/includes/_payment-method.md (hunk 1 above)
Comment (KBbitsP) [reply source/includes/_payment-method.md]: Hi @reshmabidikar made this change and added the commit.

File: source/includes/_account.md (hunk 2 above)
Comment (KBbitsP) [reply source/includes/_account.md:4439]: Hi @reshmabidikar made this change and added the commit.

**Review by reshmabidikar — APPROVED** [review]

**Pull request merged**